docker-compose.yaml

```diff
@@ -0,0 +1,27 @@
+services:
+  rabbitmq-broker:
+    image: rabbitmq:3-management
+    restart: always
+    environment:
+      RABBITMQ_DEFAULT_USER: guest
+      RABBITMQ_DEFAULT_PASS: admin
+    ports:
+      - "5672:5672"   # the port RabbitMQ will listen on
+      - "8080:15672"  # the port the dashboard web service will be available from
+
+  postgres-db:
+    build:
+      context: .
+      dockerfile: Dockerfile-db
+    container_name: postgres-db
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: postgres
```

Comment:
Is it possible to get these credentials from Env?

Reply:
Well yes, but I see no need for it since we use the docker compose only for testing and not for production.